lib/spotlight/engine.rb

```diff
@@ -305,6 +305,6 @@ def self.call(template, source = nil)
 
     config.assign_default_roles_to_first_user = true
 
-    config.exhibit_roles = %w[admin curator]
+    config.exhibit_roles = %w[admin curator viewer]
```

Comment:
There is a little confusion here by me about the relationship between this configuration `exhibit_roles` and the method `exhibit_roles`. Just confirming those are not the same?

Reply:
The method on User?

Reply:
Yes.. so `User#exhibit_roles` are roles where I can do stuff on the Exhibit. And the configuration.. does??

Reply:
Yeah; the engine configuration is the list of possible roles... and the User#exhibit_roles is the list of (curator-like) roles a user has on an exhibit. Confusing? yes... but 🤷‍♂️ ?

Reply:
Got it.. 🆗